src/pages/awsDashboard/awsDashboard.tsx

```diff
@@ -51,7 +50,7 @@ const AwsDashboardBase: React.SFC<AwsDashboardProps> = ({
         })}
       </Grid>
     ) : (
-      <Grid className={css(emptyState)} gutter="md">
+      <Grid gutter="md">
```

Comment:
Why remove the emptyState styles?

Reply:
The empty state CSS styled a `ul`, but there is no `ul` in the empty state block.

Reply:
Ok, that's fine then. I just built this locally and don't see any visual change.